lib/controller/matlab/wb_supervisor_node_set_joint_position.m

```diff
@@ -0,0 +1,6 @@
+function result = wb_supervisor_node_set_joint_position(noderef, position, index)
```

Comment:
This file should be removed because it is automatically generated.

Reply:
Yes, I just removed it.